.github/actions/pylint/index.js

```diff
@@ -0,0 +1,120 @@
+const core = require('@actions/core');
+const exec = require('@actions/exec');
+
+/**
+ * Execute pylint and report issues, if any found.
+ */
+async function run() {
+    let pylintOutput = "";  // Pylint process output.
+    const pylintDisable = core.getInput('pylint-disable') || "";
+    const pylintRCFile = core.getInput('pylint-rcfile') || "";
+    const pylintPaths = core.getInput('pylint-paths');
+
+    try {
+        // Run pylint
+        await exec.exec("pylint", [
+            `--disable=${pylintDisable}`,
+            `--rcfile=${pylintRCFile}`,
+            `--output-format=json`,
+            ...pylintPaths.split(" ")
+        ], {
+            listeners: {
+                stdout: (data) => pylintOutput += data.toString()
+            }
```

Comment:
[question] Do we ever need to worry about `stderr`?

Reply:
I don't think so. We can add that too, but I'm wary of "just in case" code. The one case I can think of that does print to stderr is if you use an unrecognized command line switch, e.g.
```
$ pylint --fgsffds 1>/dev/null
Usage: pylint [options]

pylint: error: no such option: --fgsffds
```
but we don't care about that.

Reply:
Sounds fine as is, thanks